lib/oga/xml/node.rb

```diff
@@ -55,6 +55,21 @@ def children=(nodes)
         end
       end
 
+      # Adds a child (or creates a child text node) to this node.
+      #
+      # @param [Oga::XML::Node|String] child
+
+      def <<(child)
+        child = Text.new(:text => child) if child.is_a?(String)
+
+        unless child.is_a?(Node)
+          raise ArgumentError, 'Can only add nodes or text.'
+        end
+
+        children << child
+        self
+      end
+
```

Comment:
I do not want such a method in Oga. I think it's confusing that you're calling `Node#<<` / `Node#push` to push something into its _children_, which is an entirely different data structure.

Furthermore I'm not a fan of being able to pass both a `String` and a `Node` into `child`. I already made this mistake using `Node#replace` but I'd rather not introduce any more methods doing the same.